package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "compile:sass": "node-sass sass/main.scss css/style.css -w --output-style compressed"
+    "compile:sass": "node-sass sass/main.scss css/style.css -w"
```

Comment:
I meant that usually the final resources to be used by application, which are built by tools, like css and bundled javascript, are not committed to the repository.  
In this case you could have left the minified styles if you wanted, but it's just a mention for future notice. 😉